PyDSS/value_storage.py

```diff
@@ -601,12 +601,14 @@ def append(self, values):
             list of ValueStorageBase
 
         """
+        
         if isinstance(values[0].value, list):
             vals = [x for y in values for x in y.value]
         else:
             vals = [x.value for x in values]
 
         self._dataset.write_value(vals)
+        self.old_values = values
```

Comment:
This isn’t used anywhere. Can it be deleted?

Reply:
REMOVED